plugins/github/app.ml

```diff
@@ -168,13 +168,13 @@ let app_id =
     ~docv:"ID"
     ["github-app-id"]
 
-let whitelist =
+let allowlist =
   Arg.required @@
   Arg.opt Arg.(some (list string)) None @@
   Arg.info
     ~doc:"A comma-separated list of whilelisted GitHub accounts"
```

Comment:
The documentation here should probably also be updated.

Reply:
Well spotted, thanks!